lib/ex_dtls.ex

```diff
@@ -243,6 +252,23 @@ defmodule ExDTLS do
   def handle_call(:get_cert, _from, %State{cnode: cnode} = state),
     do: {:reply, Unifex.CNode.call(cnode, :get_cert), state}
 
+  @impl true
+  def handle_info({:handle_timeout, reply_pid, timeout}, %State{cnode: cnode} = state) do
+    case Unifex.CNode.call(cnode, :handle_timeout) do
+      {:retransmit, packets} when timeout < @max_retransmit_timeout and not state.finished? ->
+        send(reply_pid, {:retransmit, 1, packets})
```

Comment:
As we disscussed we have to remove component number from here. I think we should replace it with `self()` :thinking: